### DISCUSSION TITLE
Remove trackers of Google

### DISCUSSION THREAD

**Discussion by MatadorProBr** [ReVanced/revanced-discussions#152] [Question]
The title speaks all.
I don't know if this is already a feature of Vanced so sorry if this is already being implemented.

**Comment by aaayd** [comment]
This sounds good
But this may cause an issue with things like history? 

Hopefully this can be implemented without history having issues

**Reply by u2716** [reply]
for privacy purposes a local only history would be great

**Reply by Sculas** [reply]
@Victorbry99 Does YT incognito mode not suffice for you?

**Reply by u2716** [reply]
@Sculas no, personally I don't use it. I just keep all my youtube history off,mainly because I don't want targeted ads or my yt reccomendations getting fucked up.But if there was an option for a local history (even a logfile), that doesn't go to google, I would definetily use it.Sometime it happens that I cannot find a video or a tutorial I watched a few days ago and in that case  a local history could be a life saver :)

**Reply by oSumAtrIX** [reply]
But you know to watch a video, Google has to send the video to you. They know, either way, the difference is, that it is not displayed in your history.

**Reply by Degl1** [reply]
Yeah much easier to disable your YouTube history on https://accounts.google.com/.

**Reply by u2716** [reply]
yes, they know I opened the video, but I thought that they don't collect my history data if I switch my history off.at least that's what they say.Probably I'm too naive and they collect the data anyway.
what I noticed is that with YT history off reccomendations are just random viral videos, while when it's activated, reccomandations are very targeted videos.But yes probably they have my data anyways :/

**Reply by oSumAtrIX** [reply]
It's not a question, it's inevitable unless you watch videos "without" an account associated.

**Comment by MatadorProBr** [comment]
> This sounds good But this may cause an issue with things like history?
> 
> Hopefully this can be implemented without history having issues

It should be fine if they don't block ``*s.youtube.com``

**Comment by oSumAtrIX** [comment]
> It sould be fine if they don't block `*s.youtube.com`

Not exactly how it works but removing trackers can work

**Comment by LisoUseInAIKyrios** [comment]
https://github.com/ReVanced/revanced-patches/pull/3448
